client/js/table-view.js

```diff
@@ -17,6 +17,9 @@ class TableView {
     this.headerRowEl = document.querySelector('THEAD TR');
     this.sheetBodyEl = document.querySelector('TBODY');
     this.formulaBarEl = document.querySelector('#formula-bar');
+    this.sumRowEl = document.querySelector('TFOOT');
```

Comment:
Use consistent terminology if possible. We're adding a table footer to the table, so like "this.headerRowEl" describes the header element, we could use "this.footerRowEl" to describe the same thing. Consistency within the same code contributes to better readability.